src/components/form.Select/form.Select.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
The form story should be in the main form.stories file with the other form components/